test/support/environment-variables.ts

```diff
@@ -26,6 +32,9 @@ export const getEnvironmentVariablesForProcessEnv = (): NodeJS.ProcessEnv => ({
   SINGLE_LINE_LOG_FORMAT: ENVIRONMENT_VARIABLES.SINGLE_LINE_LOG_FORMAT.toString(),
   APIM_INFORMATICA_MAX_REDIRECTS: ENVIRONMENT_VARIABLES.APIM_INFORMATICA_MAX_REDIRECTS.toString(),
   APIM_INFORMATICA_TIMEOUT: ENVIRONMENT_VARIABLES.APIM_INFORMATICA_TIMEOUT.toString(),
+  ORDNANCE_SURVEY_MAX_REDIRECTS: ENVIRONMENT_VARIABLES.ORDNANCE_SURVEY_MAX_REDIRECTS.toString(),
+  ORDNANCE_SURVEY_TIMEOUT: ENVIRONMENT_VARIABLES.ORDNANCE_SURVEY_TIMEOUT.toString(),
 });
 
 export const TIME_EXCEEDING_INFORMATICA_TIMEOUT = ENVIRONMENT_VARIABLES.APIM_INFORMATICA_TIMEOUT + 500;
+export const TIME_EXCEEDING_ORDNANCE_SURVEY_TIMEOUT = ENVIRONMENT_VARIABLES.ORDNANCE_SURVEY_TIMEOUT + 500;
```

Comment:
Why do we override some environment variables but not others? Also, it looks like the `TIME_EXCEEDING_ORDNANCE_SURVEY_TIMEOUT` references the original `ORDNANCE_SURVEY_TIMEOUT`, so won't be accurate if it is overriden.

Reply:
Not sure what you mean by override, I think we use just standard pattern in this file, I added new variables, but didn't change approach.

In function getEnvironmentVariablesForProcessEnv some overriding happens as `number` value type is changed to `string`, as all environment variables are strings. App will parse these `env` variables them back to numbers.